store/src/main/java/com/dropbox/android/external/store4/impl/RealStore.kt

```diff
@@ -245,19 +245,20 @@ internal class RealStore<Key : Any, Input : Any, Output : Any>(
                             }
                         }
                         is StoreResponse.Error -> {
-                            // disk sent an error, send it down as well
-                            emit(diskData)
```

Comment:
This is the main issue - we can't emit these errors that are coming from the SoT anymore